services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -2063,7 +2063,7 @@ public void uploadFile(UploadFileInfo uploadInfo, HttpServletRequest request) th
         final String uploadingUser = uploadInfo.getUser();
         String owner = null;
         // Owner is user if it's not a space, otherwise use space identity
-        if (!uploadInfo.getType().equals(OWNER_TYPE_SPACE) && !uploadInfo.getIdentity().equals(uploadingUser)) {
+        if (!uploadInfo.getType().equals(OWNER_TYPE_SPACEEVENT) && !uploadInfo.getIdentity().equals(uploadingUser)) {
```

Comment:
Can uploadInfo.getType() be equals to OWNER_TYPE_SPACE ? What we do in this can ?